elytron/src/test/java/org/wildfly/extension/elytron/SubsystemTransformerTestCase.java

```diff
@@ -63,14 +61,14 @@ public void testRejectingTransformersWFCore300() throws Exception {
 
         List<ModelNode> ops = builder.parseXmlResource("transformers-1.0.xml");
         PathAddress subsystemAddress = PathAddress.pathAddress(ModelDescriptionConstants.SUBSYSTEM, ElytronExtension.SUBSYSTEM_NAME);
-        ModelTestUtils.checkFailedTransformedBootOperations(mainServices, elytronVersion, ops, new FailedOperationTransformationConfig()
+        /*ModelTestUtils.checkFailedTransformedBootOperations(mainServices, elytronVersion, ops, new FailedOperationTransformationConfig()
                 .addFailedAttribute(subsystemAddress.append(PathElement.pathElement(ElytronDescriptionConstants.SIMPLE_PERMISSION_MAPPER)),
                         new MatchAllConfig()
                 )
                 .addFailedAttribute(subsystemAddress.append(PathElement.pathElement(ElytronDescriptionConstants.AUTHENTICATION_CONFIGURATION)),
                         new FailedOperationTransformationConfig.NewAttributesConfig(AuthenticationClientDefinitions.FORWARDING_MODE)
                 )
-        );
+        );*/
```

Comment:
Please restore this test and add a new one for EAP 7.1.  We may be altering code in this part of the model and I want this test to alert us in case we break things in a way we didn't expect. I don't care all that much long term about transforming to WF Core 3.0.0 but I'm not in a hurry to drop this once until the rework is done.

Reply:
That is actually it. Test was updated to properly test transformation to 7.1.
But as there are no model changes in elytron subsystem since 7.1 and master, reject check cannot reject anything (yet)

Reply:
I've pushed another commit that adds 7.1 transformers testing as well as cleaned up transformers registration for 7.1.
But as currently master == 7.1 model wise, there are no transformers and tests catch noting.